### ISSUE TITLE
Design Mic pre-amp

### ISSUE THREAD

**Issue by SmittyHalibut** [Halibut-Electronics/Open-Headset-Interconnect-Standard#3]
Design a mic pre-amp to connect a dynamic mic to device expecting an electret.

I think this can be done using the bias power on the electret mic feed:

![image](https://user-images.githubusercontent.com/497398/178335073-5fcce7ad-1f12-4c8a-a2bd-f25e92212339.png)

R2 and C2 are part of the device consuming the Electret mic.  The node between R2, C2, R1, and T1 is the "mic" pin on HIS, and ground is "mic return".  This only requires C1, R1, and T1 to turn a dynamic mic into an electret level signal.

...we just need to, ya know, pick R1 and T1.  (C1 is likely to be a 1uF ceramic.)

HIS will specify standards, but will also allow direct connection to a radio that's already expecting an elecret element.  So R2 and the voltage are likely to be all over the place.

**Comment by SmittyHalibut** [comment]
http://halibut.com/~mark/2022-08-18%20First%20Test%20Of%205%20Component%20Preamp.mp3

The pre-amp design works really well!  I've got several test boards on their way to me, and will be sending them off to users to test on their setups.  But this sounds remarkably good.

**Comment by SmittyHalibut** [comment]
Mic Preamp boards sent to users (identities not posted here, so I don't DOX them if this repo ever becomes public.)  Waiting on responses.

**Comment by SmittyHalibut** [comment]
Responses from Mic preamps have been good, with one case of noise that is likely caused by unshielded cables.

**Comment by SmittyHalibut** [comment]
![image](https://user-images.githubusercontent.com/497398/207692285-481eabeb-8ff5-459d-b047-36ad2df9903a.png)

That's from https://www.epanorama.net/circuits/dynamic_to_electretinput.html

![image](https://user-images.githubusercontent.com/497398/207692355-92c25f29-31a4-4b4f-89a9-4ac899249434.png)

And that's from https://hf5l.pl/en/electret-microphone-instead-of-dynamic-and-vice-versa/

Both are showing similar circuits, and similar to what I've already designed above.  I need to compare component values and see how close they really are.

**Comment by SmittyHalibut** [comment]
Also note https://github.com/Halibut-Electronics/Open-Headset-Interconnect-Standard/issues/6 talks about varying voltages and series resistances when driving an electret.  My original assumption about 5v and 2.2k ohms was incorrect.  So make sure the amp _I_ designed works with varying source voltages and series resistances.

The goal of the relationship between source voltage and series resistance is to drive a given bias current (0.5mA) through the electret element, and therefore through our amp circuit.  So I THINK it'll work out ok.  But I need to do the math to verify.

**Comment by SmittyHalibut** [comment]
When testing Radio Pro v1.1b in https://github.com/Halibut-Electronics/HHI-Radio-Pro/issues/19 I have been using as many mics and headsets as I can, and my Heil dynamic mic is NOT working with the mic pre-amp design I have. No audio.

I reconfigured one pre-amp as the second design in https://github.com/Halibut-Electronics/Open-Headset-Interconnect-Standard/issues/3#issuecomment-1352023799 (470k R_cb, and 100R R_e, using the 6.2k current limiting resistor in the DC bias circuit as the 5.1k R_c).  If I yell, I get a bit of heavily distorted audio.  Something isn't right there.

I think I need to simulate this circuit.  *sigh*  One of these days, I'll figure out Qucs.  Maybe this is that day?

**Comment by SmittyHalibut** [comment]
I simulated the Guitar Circuits amp, and the hf5l.pl amp.  Guitar Circuits got us about 15dB gain, and the hf5l got about 27dB.  With a 5v/6.2k power source, neither was heavily distorting, though I haven't figured out how to measure THD with Qucs yet.

**Comment by SmittyHalibut** [comment]
I'm not sure why the mic amps aren't working as expected.  I've started experimenting with OpAmp based mic amps: https://github.com/Halibut-Electronics/HHI-User-Pro/issues/10

But, this passive component amp really should work, and it did work with test boards.  But it doesn't seem to be working with the Heil headset.

**Comment by SmittyHalibut** [comment]
'CUZ I'M A DUMBASS.  When using your Heil to test Dynamic mics, make sure you have a Dynamic mic element loaded into the headset, not the iC element, meant for Icoms, which is an Electret.

Swapped the mic element, works a champ.  *sigh*  At least I learned a fair bit about circuit modeling, which is a skill I'll definitely be able to use going forward.

Also tested my PR-781 using an XLR to 3.5mm cable.  Also works great.

While working on https://github.com/Halibut-Electronics/Open-Headset-Interconnect-Standard/issues/6 I also designed an attenuator that works across both an Electret element, and the output of this dynamic mic amp.

I've moved the "Update the mic preamp design in the standards document" to https://github.com/Halibut-Electronics/Open-Headset-Interconnect-Standard/issues/10

Resolving.
